Cargo.lock

```diff
@@ -95,6 +95,18 @@ name = "ascii"
 version = "0.8.7"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 
+[[package]]
+name = "async-compression"
```

Comment:
Where's this dependency coming from? I can't seem to find what depends on it.

Reply:
`gzip` feature of `reqwest v0.10.4`